myjansson/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(myjansson)
+set(SRC
+    jansson_handle.cpp
+    myjansson.cpp
+    jansson_handle.hpp
+    myjansson.hpp
+)
+omim_add_library(${PROJECT_NAME} ${SRC})
+target_link_libraries(${PROJECT_NAME}
```

Comment:
Can we customize the 3p lib and add_subdirectory here?

Reply:
Why exactly would we do that? At first it seems contrary to the whole objective of separating cppjansson from 3party.

Reply:
The config of the 3pp lib is now in the root CMakeLists.txt.